cmake/CommonModularInterface.cmake

```diff
@@ -4,6 +4,19 @@ if(SYSTEM_INCLUDES)
 endif()
 INCLUDE_DIRECTORIES(${INCLUDES})
 
+# Generate automatic shogun-base.i from jinja2 templates
+IF(JINJA2_FOUND)
+	ADD_CUSTOM_COMMAND(OUTPUT shogun-base.i
+		COMMAND ${PYTHON_EXECUTABLE} ${COMMON_MODULAR_SRC_DIR}shogun-base.i.py
```

Comment:
i know that `${COMMON_MODULAR_SRC_DIR}` has a trailing / when it's being set but having `${COMMON_MODULAR_SRC_DIR}shogun-base.i.py` seen makes me feel... mmm so it's not looking good. does the  `${COMMON_MODULAR_SRC_DIR}/shogun-base.i.py` really hurt?